src/pages/acuity/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Script } from 'gatsby'
+import Layout from '../../components/layout/layout'
+import { SEO } from '../../components/containers'
+import { localize } from '../../components/localization'
+
+const MyComponent = () => {
+    return (
+        <Layout>
+            <SEO
+                title={localize('_t_Acuity | An Online Trading Platform | Deriv.com_t_')}
+                description={localize(
+                    '_t_Deriv is a pioneering and award-winning online trading platform that offers a wide selection of derivatives for anyone, anywhere to trade._t_',
+                )}
+            />
+            <div
+                style={{
+                    inlineSize: '100%',
+                    blockSize: '80rem',
+                    backgroundColor: 'gray',
+                }}
+            >
+                <Script
+                    id="economicCalendar"
+                    src={'https://prodstorage.azureedge.net/Widgets/lib/@1.0.0/widget-core.js'}
```

Comment:
please check what happens if you add this script to `gatsby-config.js` line : `42`. if everything is working ok, please add it there as well. thanks :pray: